src/lib/statsig/events.ts

```diff
@@ -2,4 +2,10 @@ export type Events = {
   init: {
     initMs: number
   }
+  'post:like': {
+    logContext: 'FeedItem' | 'PostThreadItem' | 'Post'
+  }
+  'post:unlike': {
+    logContext: 'FeedItem' | 'PostThreadItem' | 'Post'
```

Comment:
i guess my concern is that i do want to get more specific sometimes (like profile header) so we’ll want canonical names for these concepts — and so we might as well just use component names as first approximation.

Reply:
and then if the component structure changes sufficiently to have them renamed, that might be a good signal that we want to rename the contextual names too — since they correspond to different kind of UI elements. assuming we don’t go around renaming things for its own sake. also note that there’s no requirement the call is done in the same exact component — as long as it’s somewhere above i think the approximation is close enough.